Build Error in validate_noiseprofiles_json

Current master WSL Ubuntu build errors (clean builds by removing contents of build folder). Probably related to noise profiles added recently.

```
...
..
.

0.000367349297779409, 0.000142437567744199, 0.000285612821654188], 'b': [3.91931856021865e-07, 6.40422964348142e-07, 3.19798274358322e-07]}, {'name': 'M1 iso 12800', 'iso': 12800, 'a': [0.000646091953940536, 0.000249611704546539, 0.000503382676952126], 'b': [3.21639519553332e-07, 4.12083597021623e-07, 2.47763269471236e-07]}, {'name': 'M1 iso 25600', 'iso': 25600, 'a': [0.0015156519054789, 0.000574054729750559, 0.00117894088012683], 'b': [8.15014471836272e-07, 6.89013051723195e-07, 6.80076388708215e-07]}]}]}] is too long
[  7%] Test-compiling OpenCL program colorspace.cl
Merging translations into /home/mchhil/darktable/build/share/darktable/darktable.desktop.
Merging translations into /home/mchhil/darktable/build/bin/../share/appdata/darktable.appdata.xml.
[  7%] Built target opencl-program-rgbcurve.cl
[  7%] Built target opencl-program-nlmeans.cl
[  7%] Built target opencl-program-common.h
[  7%] Built target generate_authors_h
make[2]: *** [share/darktable/CMakeFiles/validate_noiseprofiles_json.dir/build.make:59: share/darktable/CMakeFiles/validate_noiseprofiles_json] Error 1
make[1]: *** [CMakeFiles/Makefile2:7005: share/darktable/CMakeFiles/validate_noiseprofiles_json.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[  7%] Checking validity of darktable.desktop
CREATED /home/mchhil/darktable/build/bin/../share/appdata/darktable.appdata.xml
Scanning dependencies of target opencl-program-rgblevels.cl
Scanning dependencies of target opencl-program-bloom.cl
Scanning dependencies of target opencl-program-colorreconstruction.cl
Scanning dependencies of target opencl-program-dwt.cl
[  7%] Built target darktable.appdata_file
[  7%] Built target validate_darktable_desktop
[  7%] Test-compiling OpenCL program bloom.cl
[  7%] Test-compiling OpenCL program rgblevels.cl
[  7%] Test-compiling OpenCL program colorreconstruction.cl
[  7%] Test-compiling OpenCL program dwt.cl
[  7%] Built target opencl-program-colorspace.cl
[  7%] Built target opencl-program-rgblevels.cl
[  7%] Built target opencl-program-bloom.cl
[  7%] Built target opencl-program-colorreconstruction.cl
[  7%] Built target opencl-program-dwt.cl
make: *** [Makefile:152: all] Error 2
```

--- a/data/noiseprofiles.schema
+++ b/data/noiseprofiles.schema
@@ -24,6 +24,7 @@
             "enum": [
               "Canon",
               "Fujifilm",
+              "Hasselblad",
               "LGE",
               "Leica",
               "Minolta",

